src/styling.rs

```diff
@@ -0,0 +1,613 @@
+            '0'..='9' => 0x1D7BC,
+            _ => return c,
+        };
+        apply_delta(c, delta)
+    }
+
+    /// To sans-serif italic symbols.
+    pub fn to_sans_serif_italic(c: char) -> char {
+        let delta = match c {
+            'A'..='Z' => 0x1D5C7,
+            'a'..='z' => 0x1D5C1,
+            _ => return c,
+        };
+        apply_delta(c, delta)
+    }
+
+    /// To sans-serif bold italic symbols.
+    pub fn to_sans_serif_italic_bold(c: char) -> char {
+        let delta = match c {
+            'A'..='Z' => 0x1D593,
+            'a'..='z' => 0x1D58D,
+            'Α'..='Ρ' => 0x1D3C5,
+            'ϴ' => 0x1D373,
+            'Σ'..='Ω' => 0x1D3C5,
+            '∇' => 0x1B568,
+            'α'..='ω' => 0x1D3BF,
+            '∂' => 0x1B587,
+            'ϵ' => 0x1D395,
+            'ϑ' => 0x1D3BA,
+            'ϰ' => 0x1D39C,
+            'ϕ' => 0x1D3B8,
+            'ϱ' => 0x1D39D,
+            'ϖ' => 0x1D3B9,
+            '0'..='9' => 0x1D7BC,
+            _ => return c,
+        };
+        apply_delta(c, delta)
+    }
+
+    /// To fraktur symbols.
+    ///
+    /// This style is sometimes known as black-letter.
+    pub fn to_fraktur(c: char) -> char {
+        let delta = match c {
+            'C' => 0x20EA,
+            'H' => 0x20C4,
+            'I' => 0x20C8,
+            'R' => 0x20CA,
+            'Z' => 0x20CE,
+            'A'..='Z' => 0x1D4C3,
+            'a'..='z' => 0x1D4BD,
+            _ => return c,
+        };
+        apply_delta(c, delta)
+    }
+
+    /// To bold fraktur symbols.
+    pub fn to_fraktur_bold(c: char) -> char {
+        let delta = match c {
+            'A'..='Z' => 0x1D52B,
+            'a'..='z' => 0x1D525,
+            _ => return c,
+        };
+        apply_delta(c, delta)
+    }
+
+    /// To script symbols.
+    pub fn to_script(c: char) -> char {
+        let delta = match c {
+            'B' => 0x20EA,
+            'E'..='F' => 0x20EB,
+            'H' => 0x20C3,
+            'I' => 0x20C7,
+            'L' => 0x20C6,
+            'M' => 0x20E6,
+            'R' => 0x20C9,
+            'A'..='Z' => 0x1D45B,
+            'e' => 0x20CA,
+            'g' => 0x20A3,
+            'o' => 0x20C5,
+            'a'..='z' => 0x1D455,
+            _ => return c,
+        };
+        apply_delta(c, delta)
+    }
+
+    /// To bold script symbols.
+    pub fn to_script_bold(c: char) -> char {
+        let delta = match c {
+            'A'..='Z' => 0x1D48F,
+            'a'..='z' => 0x1D489,
+            _ => return c,
+        };
+        apply_delta(c, delta)
+    }
+
+    /// To chancery style of script symbols.
+    ///
+    /// This function returns a variation sequence using the variation selector
+    /// `U+FE00`, as specified in the [`StandardizedVariants.txt`] file from
+    /// the Unicode Character Database. Note that only the capital latin
+    /// letters are standardized variation sequences, but this function also
+    /// maps the small latin letters.
```

Comment:
```suggestion
    /// the Unicode Character Database. Note that only the capital Latin
    /// letters are standardized variation sequences, but this function also
    /// maps the small Latin letters.
```

And similar in three other places.